home.php

```diff
@@ -1,5 +1,4 @@
-<html>
-   
+
```

Comment:
This doesn't need to get in the way of getting this deployed, but you normally want to avoid having whitespace like this before the `<?php`. It's rare that it matters, but it can cause weird unexpected behavior sometimes.